src/main/java/de/retest/recheck/report/ActionReplayResult.java

```diff
@@ -246,7 +246,7 @@ public Screenshot getTargetScreenshot() {
 	}
 
 	public int getCheckedUiElementsCount() {
-		if ( !windows.isEmpty() ) {
+		if ( windows != null && !windows.isEmpty() ) {
```

Comment:
How can this happen anyways? The only way is through deserialization with `ActionReplayResult()` as the other constructor should prevent passing nulls.

Also, down below there is a method `hasWindows()`, if you like to use it.